doc/whatsnew/fragments/2832.feature

```diff
@@ -0,0 +1,6 @@
+Checks for generators that use contextmanagers that don't handle cleanup properly.
```

Comment:
(FYI, if you use the `new_check` suffix instead of `feature` it will be grouped a little bit more helpfully.)